src/multichain_demo.js

```diff
@@ -0,0 +1,1151 @@
+import * as d3 from "d3";
+import { Contract } from 'web3-eth-contract';
+import abi from './multichain_demo_contract_abi.json'
+const bridgeContract = new Contract(abi)
+
+//
+// Constants
+//
+const WeiPerEth = 1000000000000000000
+
+const BridgeableScopes = {
+    // Sepolia
+    "eip155:11155111": {
+        name: "Sepolia",
+        contractAddress: '0x2e2512fd69cba059DFf557cD6f683a3279402e91',
```

Comment:
note you will have to deploy your own dummy bridge contracts to each of these testnets as the `withdraw` method is locked to my wallet addresses for the currently deployed contracts listed in this constant. The solidity file is in this PR. Note that there is no actual bridging, just piggy bank contracts that mock the bridging flow that is:
1. sending to originating chain bridge contract
2. claiming on target chain bridge contract

Reply:
BTW I used the online Remix IDE to compile and deploy. EVM version: Paris (because of Linea not supporting an opcode)